src/descriptors/LITS-full-structs.json

```diff
@@ -0,0 +1,1082 @@
+[
```

Comment:
Should this file be called LITS_structs.json?

Reply:
This file got removed